docker/is_bot_online.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+# Copyright 2019 Semyon Suprun <simonsuprun@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from optparse import OptionParser
+from sys import stderr
+from time import sleep
+
+from base import SplinterTestCase
+
+
+LOCALHOST = 'http://127.0.0.1:8006'
+
+
+class IsBotOnline(SplinterTestCase):
+    def __init__(self, addr, username, password, **kwargs):
+        SplinterTestCase.__init__(self, addr)
+
+        self.username = username
+        self.password = password
+        self.bot_name = kwargs.get('bot')
+        self.wait = kwargs.get('wait')
+
+    def test_is_bot_is_online(self):
+        self.browser.fill('emailOrUsername', self.username)
+        self.browser.fill('pass', self.password)
+        self.find_by_css('.rc-button.rc-button--primary.login').click()
+
+        search_btn = self.find_by_css(
+            '.rc-icon.sidebar__toolbar-button-icon'
+            '.sidebar__toolbar-button-icon--magnifier'
+        )
+        assert search_btn
+        search_btn.click()
+
+        search = self.browser.find_by_css('.rc-input__element')
+        assert search
+        search.fill(self.bot_name)
+
+        channels = self.browser.find_by_css('.sidebar-item.popup-item')
+        assert channels
+        channels.first.click()
+
+        is_online = False
+        for _ in range(int(self.wait)):
+            status = self.find_by_css('.rc-header__visual-status')
+            if status.first.text.lower() == 'offline':
+                sleep(1)
+            else:
+                is_online = True
+                break
+        assert is_online
+
+
+def main():
+    parser = OptionParser(usage='usage: %prog [options] arguments')
```

Comment:
optparse is deprecated since version Python 3.2.
I understand that the project currently uses optparse, but we have to get rid of it. I've just created the [issue](#77).